README.md

```diff
@@ -119,6 +122,16 @@ The set of attributes the user has configured for the transaction.
 #### YotiClient
 
 Allows your app to retrieve a user profile, given an encrypted token.
+
+## How to run the examples
+
+The examples can be found in the [examples folder](https://github.com/getyoti/php/tree/master/example) which requires few setups to work as explained below.
```

Comment:
You don't need the full URL. It can be [example folder](example). I would write it as: The example can be found in the [example folder](example). The steps required for the setup are explained below.